pkg/hostagent/hostagent_unix.go

```diff
@@ -0,0 +1,28 @@
+//go:build !windows
```

Comment:
This line is probably not needed for recent Go

Reply:
I tried building with go 1.21 and got the error `adjustNofileRlimit redeclared in this block`, so I added a limit

Reply:
I think, there should be

```
//go:build unix
```

because file suffix `_unix` is not working in Go:
https://github.com/golang/go/issues/51572

still, for file `hostagent_windows` build tag is not required, as suffix `_windows` works well